Content.Client/Bank/BUI/WithdrawlBankATMMenuBoundUserInterface.cs

```diff
@@ -0,0 +1,51 @@
+using Content.Client.Bank.UI;
+using Content.Shared.Bank.BUI;
+using Content.Shared.Bank.Events;
+using Robust.Client.GameObjects;
+
+namespace Content.Client.Cargo.BUI;
+
+public sealed class WithdrawBankATMMenuBoundUserInterface : BoundUserInterface
+{
+    private WithdrawBankATMMenu? _menu;
+
+    public WithdrawBankATMMenuBoundUserInterface(ClientUserInterfaceComponent owner, Enum uiKey) : base(owner, uiKey) {}
+
+    protected override void Open()
+    {
+        base.Open();
+
+        _menu = new WithdrawBankATMMenu();
+        _menu.WithdrawRequest += OnWithdraw;
+        _menu.OnClose += Close;
+        _menu.OpenCentered();
+    }
+
+    protected override void Dispose(bool disposing)
+    {
+        base.Dispose(disposing);
+        if (disposing)
+        {
+            _menu?.Dispose();
+        }
+    }
+
+    private void OnWithdraw()
+    {
+        if (_menu?.Amount is not int amount)
+            return;
+
+        SendMessage(new BankWithdrawMessage(amount));
```

Comment:
our server side system does indeed validate entries before accepting along with providing player feedback so this is fine without a validation check for now